src/util/std_expr.cpp

```diff
@@ -43,6 +43,58 @@ exprt conjunction(const exprt::operandst &op)
   }
 }
 
+// Create a lexicographic less-than relation between two variables.
+// This is used in the implementation of multidimensional decreases clauses.
+exprt create_lexicographic_less_than(
+  const std::vector<symbol_exprt> &small,
+  const std::vector<symbol_exprt> &large)
+{
+  PRECONDITION(small.size() == large.size());
+
+  if(small.size() == 0)
+  {
+    return true_exprt();
```

Comment:
Shouldn't this be `false_exprt()`?

Empty tuple is not _strictly_ less than another empty tuple.

Reply:
Thanks for spotting a bug! I agree that the output should be false.